main/src/ca/uwaterloo/flix/api/lsp/provider/FindReferencesProvider.scala

```diff
@@ -15,12 +15,14 @@
  */
 package ca.uwaterloo.flix.api.lsp.provider
 
-import ca.uwaterloo.flix.api.lsp.{Entity, Index, Location, Position}
+import ca.uwaterloo.flix.api.lsp.{Entity, Index, Location, LocationLink, Position}
 import ca.uwaterloo.flix.language.ast.TypedAst.{Expression, Pattern, Root}
 import ca.uwaterloo.flix.language.ast.{Name, Symbol, TypeConstructor}
 import org.json4s.JsonAST.JObject
 import org.json4s.JsonDSL._
 
+import scala.::
```

Comment:
This is always imported. No need for an explicit import :)

Reply:
You are right sorry!